contrib/fuzz/archive_fuzzer.go

```diff
@@ -73,6 +77,31 @@ func FuzzImportIndex(data []byte) int {
 	}
 	ctx := context.Background()
 	r := bytes.NewReader(tarBytes)
+	doThis, err := f.GetBool()
```

Comment:
Can we have a more descriptive variable name?  `shouldRequireLayoutOrManifest` maybe?